Sources/PackageModel/Module.swift

```diff
@@ -129,9 +130,24 @@ public class ClangModule: CModule {
     }
 }
 
+/// A dictionary containing mapping of C family source extension to their file type in Xcode.
+private let xcodeCLanguageFileTypeDict = [
+    ".c": "sourcecode.c.c",
+    ".m": "sourcecode.c.objc",
+    ".cxx": "sourcecode.cpp.cpp",
+    ".cc": "sourcecode.cpp.cpp",
+    ".cpp": "sourcecode.cpp.cpp",
+    ".mm": "sourcecode.cpp.objcpp",
+]
+
 extension ClangModule: XcodeModuleProtocol {
-    public var fileType: String {
-        return "sourcecode.c.c"
+    public func fileType(forSource source: String) -> String {
```

Comment:
LGTM but small note. 
source  - `someFile.c`

Would it be better to get file extension part and check for that key in the `xcodeCLanguageFileTypeDict ` ?

```swift
public func fileType(forSource source: String) -> String {
 let fileExtension =  source.fileExtension
 guard let fileType = xcodeCLanguageFileTypeDict[fileExtension] else {
    fatalError
}
return fileType
}
```